.travis.yml

```diff
@@ -5,6 +5,9 @@ before_script:
   - psql -c 'create database dummy_qc_admin_test;' -U postgres
   - "bundle exec rake db:schema:load db:migrate RAILS_ENV=test"
 rvm:
-  - 2.7.2
-  - 2.6.6
-  - 2.5.8
+  - 3.3.1
+  - 3.2.4
+  - 3.1.5
+  - 3.0.7
+  - 2.7.8
+  - 2.6.10
```

Comment:
These are all EOL so I think it's fine to remove them from the test matrix?

Reply:
@magni- the "rule" for the main QC repo is just to support what's current (ish) ruby wise - e.g. https://github.com/QueueClassic/queue_classic/pull/347 - seems kinda sane for this project too?